dreem/datasets/data_utils.py

```
@@ -11,6 +11,8 @@
 import pandas as pd
 import sleap_io as sio
 import torch
+from dreem.io import Instance
+from dreem.io import Instance
```

Ruff (0.8.2)

Redefinition of unused `Instance` from line 14

Remove definition: `Instance`

(F811)